.github/workflows/test.yml

```diff
@@ -174,10 +134,3 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node }}
-
-      # Delete the cache so it is only used once
-      - name: Delete Cache
```

Comment:
no longer needed since we don't need the build files to be cached between the steps